Core/Core/Router/Router.swift

```diff
@@ -180,6 +180,7 @@ public class Router: RouterProtocol {
         #if DEBUG
         DeveloperMenuViewController.recordRouteInHistory(url.url?.absoluteString)
         #endif
+        Analytics.shared.logEvent("route", parameters: ["path": url.path])
```

Comment:
would it be more helpful to have the whole url? We sometimes use `?query` params or follow the `#fragment`, it might also be interesting how often the url contains the host.

Reply:
Yeah might as well!